### ISSUE TITLE
Can I add the contributors pictures below your picture?

### ISSUE THREAD

**Issue by sansyrox** [charmaine/InternetWishlist#38]
@charmaine 
Can I add the pictures of contributors below your photo to acknowledge their work?

**Comment by charmaine** [comment]
YES please! Feel free to add a note in the README to remind others to add their pics before they submit a PR too! 🎉

**Comment by maitreyav** [comment]
I was also thinking to do that. I feel, we should make it  more interactive and a PWA to reach out to users. What you say?

**Comment by charmaine** [comment]
Hey @maitreyav! That would be awesome-- I quite like PWAs myself. Let me know if you are interested in kickstarting the transition to a PWA. 😊

**Comment by maitreyav** [comment]
@charmaine, Good to know that! 👍  Yes, I'm interested to see where it can go.

**Comment by maitreyav** [comment]
@stealthanthrax , How are you planning to do it? Through an API call?

**Comment by sansyrox** [comment]
@maitreyav i plannig to use the github images as the pictures.
@charmaine there are 19 contributors,so is there a specific way in which they should be ordered??

**Comment by charmaine** [comment]
@stealthanthrax Hey! We can list them in chronological order based on when they first contributed to the project (oldest first)? Let me know if you agree with the logic there!

**Comment by sansyrox** [comment]
@charmaine I agree with you .There's another suggestion that we can also identify a monthly star performer based on the maximum contribution . 😄

**Comment by maitreyav** [comment]
@stealthanthrax, Check this out. 
https://github.com/mgechev/github-contributors-list

**Comment by charmaine** [comment]
@maitreyav, great find! @stealthanthrax It looks like that would be a good solution! 👍
